### PR TITLE
Parallelize function visual

### PR DESCRIPTION
**Pull request by oHunewald**

### CODE REVIEW

File: test/parallel.jl [hunk 1]
```diff
@@ -26,11 +26,9 @@ som2 = initGigaSOM(dfSom, 10, 10)
     @test som2.numCodes == 100
 end
 
-som2 = trainGigaSOM(som2, dfSom, epochs = 2, r = 6.0)
+@time som2 = trainGigaSOM(som2, dfSom, epochs = 2, r = 6.0)
```
Comment (laurentheirendt) [inline test/parallel.jl]: I think there is no real need for `@time` here.

File: test/parallel.jl [hunk 2]
```diff
@@ -26,11 +26,9 @@ som2 = initGigaSOM(dfSom, 10, 10)
...
 
-winners = mapToGigaSOM(som2, dfSom)
-
-winners = mapToGigaSOM(som2, dfSom)
+@time winners = mapToGigaSOM(som2, dfSom)
```
Comment (laurentheirendt) [inline test/parallel.jl]: Same comment here

File: src/core.jl [hunk 3]
```diff
@@ -201,16 +197,40 @@ every row in data.
...
+    nWorkers = nprocs()
+    dData = distribute(data)
+    vis = Int64[]
+
+    if nWorkers > 1
+        # distribution across workers
+        R = Array{Future}(undef,nWorkers, 1)
+         @sync for p in workers()
+
+             println("worker: $p")
```
Comment (laurentheirendt) [inline src/core.jl:214]: Would it make sense to introduce here a `printLevel` variable?

**Review by laurentheirendt — CHANGES_REQUESTED** [review]

File: src/core.jl (hunk 3 above)
Comment (oHunewald) [reply src/core.jl:214]: I'll take this out and only print the radius...

**Review by laurentheirendt — APPROVED** [review]

**Pull request merged**